src/hooks/useTokenBalance.ts

```diff
@@ -61,4 +63,24 @@ export const useBurnedBalance = (tokenAddress: string) => {
   return balance
 }
 
+export const useGetBnbBalance = () => {
+  const [balance, setBalance] = useState(new BigNumber(0))
+  const { account } = useWeb3React()
+  const { lastUpdated, setLastUpdated } = useLastUpdated()
+
+  useEffect(() => {
+    const fetchBalance = async () => {
+      const web3 = getWeb3NoAccount()
```

Comment:
Shouldn't this be `const web3 = useWeb3()`?

Reply:
Yeah...why does this work? :)

Reply:
@Chef-Cheems updated

Reply:
@hachiojidev you forgot to remove getWeb3NoAccount import :)
Yeah regarding how it works.... I dunno, I guess Web3NoAccount is still valid Web3 that can query account balances via address, so in this case it doesn't really mater? 🤔 
I haven't looked that deep into web3 setup in PCS yet but some stuff seems a bit weird for me, there might be some stuff to optimize or at least to document 😅